atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/SweepStatsKeyValueService.java

```diff
@@ -84,8 +82,7 @@
 
     private final Multiset<TableReference> writesByTable = ConcurrentHashMultiset.create();
 
-    private final Set<TableReference> clearedTables = Collections.newSetFromMap(
-            new ConcurrentHashMap<TableReference, Boolean>());
+    private final Set<TableReference> clearedTables = Sets.newConcurrentHashSet();
```

Comment:
Exact same thing underneath the hood at the moment, but we could pick up future improvements automatically, and this prevents someone else from digging in to the same thing.